===================================================================
RCS file: /opt/sourcecast/data/ccvs/repository/l10n/i18npool/source/localedata/data/tr_TR.xml,v
retrieving revision 1.7
retrieving revision 1.7.130.1
diff -u -r1.7 -r1.7.130.1
--- i18npool/source/localedata/data/tr_TR.xml	2003/04/08 15:58:56	1.7
+++ i18npool/source/localedata/data/tr_TR.xml	2004/10/13 15:38:38	1.7.130.1
@@ -342,10 +342,17 @@
 </Calendar>
 </LC_CALENDAR>
 <LC_CURRENCY>
-<Currency default="true" usedInCompatibleFormatCodes="true">
+<Currency default="false" usedInCompatibleFormatCodes="true">
 <CurrencyID>TL</CurrencyID>
 <CurrencySymbol>TL</CurrencySymbol>
 <BankSymbol>TRL</BankSymbol>
+<CurrencyName>Lira</CurrencyName>
+<DecimalPlaces>2</DecimalPlaces>
+</Currency>
+<Currency default="true" usedInCompatibleFormatCodes="false">
+<CurrencyID>TRY</CurrencyID>
+<CurrencySymbol>YTL</CurrencySymbol>
+<BankSymbol>TRY</BankSymbol>
 <CurrencyName>Lira</CurrencyName>
 <DecimalPlaces>2</DecimalPlaces>
 </Currency>
